example-problems/class-removed-from-google-http-client/README.md

```diff
@@ -0,0 +1,134 @@
+This project demonstrates a hidden incompatibility between
+`com.google.api-client:google-api-client:1.27.0` and
+`com.google.cloud:google-cloud-bigtable:0.81.0-alpha`. The compilation fails due to the
```

Comment:
The compilation of what? This probably belongs later in the document. The first sentence is enough summary.

Reply:
Moved this compilation failure part to later part.